Library/Homebrew/cmd/info.rb

```diff
@@ -54,10 +55,15 @@ def print_info
           else
             info_formula Formulary.find_with_priority(f)
           end
-        rescue FormulaUnavailableError
+        rescue FormulaUnavailableError => e
           # No formula with this name, try a blacklist lookup
-          raise unless (blacklist = blacklisted?(f))
-          puts blacklist
+          if (blacklist = blacklisted?(f))
+            ofail "#{e.message}\n#{blacklist}"
+          else
+            ofail e.message
+            ohai "Searching for migrated formulae..."
```

Comment:
How long does `search_for_migrated_formula` take to run? I wonder if we can skip this message.